.github/workflows/cgroup2.yaml

```diff
@@ -16,10 +16,12 @@ jobs:
       fail-fast: false
       matrix:
         provider: [docker, podman]
+        rootless: ["rootful", "rootless"]
```

Comment:
I'm not English native, but I think it is named `rootfull` with an additional `l`.
I wouldn't mind about it, but since there are always "typo hunters", if we correct it later, this will change the name of the job later

Reply:
I'm not an English native either, but according to wiktionary, "-ful" is correct.
https://en.wiktionary.org/wiki/-ful#English

Reply:
ah, thanks, always learning something new :)